### ISSUE TITLE
Broken Journalism

### ISSUE THREAD

**Issue by allenleein** [allenleein/brains#1321]
I can't think of a job where the deep understanding of ethics is more important than journalism.

Who reported the 2008 financial crisis/ ISIS / War of Syria? Some journalists did that. Who reported the fake news/gossip from Youtube and Instagram? Some journalists did that, too. One man was responsible for spreading information that inspires tens of thousands of lives. Another man helped make millions more superficial as thinkers.

In short, social media's effect on journalism is much greater than the shift from print to digital. Another shift is in journalism from reporting facts to "get as many pageviews as you can", driven by the fact that we give up our attention and our privacy rather than our money to the REAL publisher.
